### PR TITLE
[strings][categories] Updated Italian translations

### PR DESCRIPTION
**Pull request by FinixFighter**
Signed-off-by: Luca Finizio <luca.finizio@protonmail.com>

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -5240,7 +5240,7 @@ fr:Vêtement usagés|Déposer des affaires non-utilisées
 de:4Altkleider|Alte Kleidung|Sachen spenden
 hu:Régi ruhák|Régi ruhák újrahasznosítása
 id:Pakaian lama|Mendaur ulang pakaian lama
-it:Vestiti usati|Riciclare vestiti usati
+it:Vestiti usati|Riciclare vestiti usati|Raccolta indumenti
```
Comment (biodranik) [inline data/categories.txt:5243]: I would add a number like 4 or 5 before starting words to show hints. Are you sure that someone guesses and types all these letters exactly, as they are written here?

Comment (FinixFighter) [reply data/categories.txt:5243]: > I would add a number like 4 or 5 before starting words to show hints.

I'm sorry, I don't remember what 4 or 5 numbers mean?

>Are you sure that someone guesses and types all these letters exactly, as they are written here?

Yes I'm sure. In fact, that's why I made this edit. I searched for "raccolta indumenti" on Organic Maps but results weren't exact. Then I thought that maybe this sentence wasn't used, so I checked and added it. You can also find photos of recycling containers for clothes with "raccoltaindumenti" printed on (for instance: https://www.vastoweb.com/news/attualita/757746/annullata-temporaneamente-la-procedura-di-gara-per-la-raccolta-indumenti-usati)

Comment (biodranik) [reply data/categories.txt:5243]: Please check the beginning of this file, it describes in details what these number prefixes mean.

Comment (FinixFighter) [reply data/categories.txt:5243]: > Please check the beginning of this file, it describes in details what these number prefixes mean.

Yes I read it, and I edited my message. I agree about adding a 4 before starting words.

Comment (biodranik) [reply data/categories.txt:5243]: Then you should carefully review all other recycling keywords. If you add numbers only to the clothes, then other recycling types won't be hinted.

**Review by biodranik — APPROVED** [review]
I'm merging it without hints, you can always add them and test later to improve UX.

File: data/categories.txt (hunk 1 above)
Comment (FinixFighter) [reply data/categories.txt:5243]: Sure! I'll check them

**Pull request merged**